modules/ROOT/pages/dw-operators.adoc

```diff
@@ -336,3 +336,30 @@ output application/json
   ]
 }
 ----
+
+== Scope and Flow Control Operators
+
+DataWeave 2.0 supports operators that control the flow and scope of expressions:
+
+* <<control_flow_do,`do` and `using`>>
+* <<control_flow_if,`if else` and `else if`>>
+
+[[control_flow_do]]
+.Scope Operators:
+[%header,cols="1,2"]
+|===
+| Operator | Description
+| `do` | Creates a scope in which new variables, functions, annotations, or namespaces can be declared and used. The syntax is similar to a mapping in that it is composed of a header and body separated by `&#45;&#45;&#45;`. Its header is where all the declarations are defined, and its body is the result of the expression. For examples, see xref:dataweave-flow-control.adoc#control_flow_do[do] and
```

Comment:
"For examples, see do? and Examples: Local DataWeave Variables"?

Reply:
revised